ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -266,6 +266,32 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: cluster-api
+  cron: 20 13 * * Mon,Thu
+  steps:
+    test:
+    - as: cluster-api
+      commands: |
+        rebasebot --source https://github.com/kubernetes-sigs/cluster-api:main \
```

Comment:
What if we got the branch of the latest release like so:
```
$ curl --silent "https://api.github.com/repos/kubernetes-sigs/cluster-api/releases/latest" |
  grep '"tag_name":' |
  sed -E 's/.*"([^"]+)".*/\1/' |
  sed -E 's/^v?([0-9]+\.[0-9]+)\.[0-9]+$/release-\1/'

# input: v1.3.2  
# output: release-1.3
```
and used that as the source to pin the rebase to the latest, most stable branch?